config/api.yml

```diff
@@ -967,6 +967,8 @@
         :identifier: conversion_host_edit
       - :name: delete
         :identifier: conversion_host_delete
+      - :name: resource
+        :identifier: conversion_host_resource
```

Comment:
you are doing a `get`, right?

Reply:
Can't do custom actions with a `GET` in the ManageIQ rest api. Not as far as I can tell, as it failed when I tried, and the docs say that this is what you're supposed to do:

https://github.com/ManageIQ/manageiq-api/blob/master/CONTRIBUTING.md#custom-resource-actions

@abellotti That's correct, right?